cibyl/models/ci/zuul/job.py

```diff
@@ -123,6 +123,16 @@ def __eq__(self, other):
 
     API = {
         **subset(BaseJob.API, ['name', 'url', 'builds']),
+        'output_mode': {
+            'arguments': [
+                Argument(
+                    name='--complete', arg_type=str, nargs=0, default=False,
```

Comment:
Just for consistency sake, 'arg_type' should be set to 'bool'. 

I am also not even sure if the argument needs a default value or not, because what you are interested in is whether the argument is present. '--last-build' is a similar argument to this one, as it does not take a value, you could take a look at how that one was done.

Reply:
Good tip. I changed it to bool and did not pass the value, only if the argument is present or not